utilities/taxonomyMaps.ts

```diff
@@ -15,3 +15,17 @@ export const initiativesMap = {
   'undergraduate-financial-aid': 'Undergraduate Financial Aid',
 };
 export type InitiativesType = keyof typeof initiativesMap;
+
+export const themesMap = {
+  'accelerating-solutions': 'Accelerating Solutions',
+  'catalyzing-discovery': 'Catalyzing Discovery',
+  'preparing-citizens': 'Preparing Citizens',
+  'sustaining-life': 'Sustaining Life',
+};
```

Comment:
So all of these maps are copies of the key/label pairs in the datasource config?

I'm not crazy about duplicating and hard coding these values here as ideally they would/could be edited like content in the CMS. If we cannot pull and infer the types I'd rather see these set to string.

Reply:
`{[key:string]: string}[]`

Reply:
https://bobbyhadz.com/blog/typescript-key-string-string

Reply:
Yeah I can see pulling from datasource might be better. Let me look into server side fetching for these 🤔

Reply:
@sherakama - I'm going to add a TODO here for fetching server side after I come back. I'll create a ticket so it won't be an empty promise 😁 After that I'm going to merge this branch into the integration branch just so I can keep rolling things in before I go on vacay.

Reply:
Ticket created here:
https://stanfordits.atlassian.net/browse/GIVCAMP-355